components/NewsItem.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+	<!-- This compenent is used by data from pages api call as well as post api call -->
+	<nuxt-link :to="'/news-and-views/' + slug" class="news-item">
```

Comment:
Make the containing element an `<article>`, put the link inside the `<h2>` surrounding the title.